### PR TITLE
[bitnami/kafka,kube-prometheus,kubernetes-event-exporter] Replace K8s List

### LINKED ISSUE #4559: K8s List: Helm linter fails due to missing name
**Which chart**:
kafka
kube-prometheus/templates
kubernetes-event-exporter

**Describe the bug**

Yesterday I upgraded to the latest helm version (v3.4.1) and tried to lint a chart I wrote myself, which contains a Kubernetes list. Suddenly I got this linting error:

```
templates/list.yaml: object name must be between 0 and 253 characters: ""
```

This relates to this [Issue](https://github.com/helm/helm/issues/9019) or atleast someone else is having the same problems. I figured that this error occured, because my list resource didn't have any metadata at all (because it's not really of any use). This is what the list looked like while getting this error:

```
apiVersion: v1
kind: List
items:
  ....
```
After adding a name, the lint won't fail anymore:

```
apiVersion: v1
kind: List
metadata:
  name: "list"
items:
  ....
```

I was thinking of the kafka chart, since I know that it also implements kubernetes list. So I wanted to bring this to your attention. I made i quick grep over the chart directory and discovered, that currently three charts implement lists without having a name:

```
$ grep -R "kind: List" bitnami/*
  bitnami/kafka/templates/extra-list.yaml:kind: List
  bitnami/kube-prometheus/templates/prometheus/additionalPrometheusRules.yaml:kind: List 
  bitnami/kubernetes-event-exporter/templates/extra-list.yaml:kind: List
```

Now i am not quiet sure which approach you would chose to fix it. I see other charts with an `èxtraResource``functionality don't implement a kubernetes list (eg.  https://github.com/bitnami/charts/blob/master/bitnami/airflow/templates/extradeploy.yaml). Changing the named charts to this functionality would already resolve the issue. Or you could just simple give the lists a name, as far as I am aware this doesn't affect any of the items or anything else. It's just the helm validation that throws this error.

**To Reproduce**

1. Install the latest helm release
2. Create a chart and add a simple list:

```
apiVersion: v1
kind: List
items:
  - apiVersion: v1
    kind: ServiceAccount
    metadata:
      name: build-robot
    automountServiceAccountToken: false
```
3. Try to lin it (`helm lint .`)

**Expected behavior**

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.4.1", GitCommit:"c4e74854886b2efe3321e185578e6db9be0a6e29", GitTreeState:"clean", GoVersion:"go1.14.11"}

```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"archive", BuildDate:"2020-11-15T17:26:49Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.9+k3s1", GitCommit:"630bebf94b9dce6b8cd3d402644ed023b3af8f90", GitTreeState:"clean", BuildDate:"2020-09-17T19:05:07Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**Additional context**

Before I made a contribution I wanted to ask for your opinion on this.

### PR DESCRIPTION
**Pull request by oliverbaehler**
**Description of the change**

Replaces Kubernetes List Resources with YAML rendered List. This PR changes all resources of Kind List in the bitnami chart repository to instead render a YAML list. The change does not implement or change any values, just how raw elements (`extraDeploys`) are rendered.

**Benefits**

Mainly fixes the error which is described in the issue but also improves consistency. Now all the charts don't use K8s Lists anymore.

**Possible drawbacks**

Don't see any yet.

**Applicable issues**

Please refer to this issue
  - fixes #4559

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/kafka/Chart.yaml [hunk 1]
```diff
@@ -29,4 +29,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.2.1
+version: 12.2.2
```
Comment (FraPazGal) [inline bitnami/kafka/Chart.yaml:32]:
```suggestion
version: 12.2.4
```

There has been a couple of PRs on this chart on the last 24h and the current released version is `12.2.3`.

**Review by FraPazGal — CHANGES_REQUESTED** [review]
Hi @oliverbaehler,

Thank you so much for taking the time to fix this!! Just a small comment, please take a look at it.

File: bitnami/kafka/Chart.yaml (hunk 1 above)
Comment (oliverbaehler) [reply bitnami/kafka/Chart.yaml]: @FraPazGal Thanks for your review. Incremented Kafka Helm Chart version to `12.2.4`

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**